src/lib/autocomplete/autocomplete.ts

```diff
@@ -0,0 +1,11 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-autocomplete',
```

Comment:
Also add `mat-autocomplete` and add the appropriate selectors under `core/compatibility`